### PR TITLE
V3/misc

### PR DESCRIPTION
**Pull request by abea**
Migrates additional Storybook stories

### CODE REVIEW

**Comment by abea** [comment]
I did update the FilterMenu story since the `menu` prop was changed to `filters` and is structurally different. I also went with this `getData` function patter instead of having a separate file with story data so we could still have stories for some of these molecule-level components.

File: lib/modules/@apostrophecms/ui/src/apos/components/AposAreaMenu.vue [hunk 1]
```diff
@@ -0,0 +1,295 @@
...
+      if (this.$refs[name]) {
+        this.$refs[name][0].querySelector('button').focus();
+      }
+    }
+  }
+};
+</script>
+
+<style lang="scss" scoped>
+// @import '../../scss/_mixins';
```
Comment (gregvanbrug) [inline lib/modules/@apostrophecms/ui/src/apos/components/AposAreaMenu.vue:184]: This should be able to be safely removed.

File: lib/modules/@apostrophecms/ui/src/apos/components/AposAreaMenuItem.vue [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+  methods: {
+    click() {
+      this.$emit('click');
+    }
+  }
+};
+</script>
+
+<style lang="scss" scoped>
+// @import '../../scss/_mixins';
```
Comment (gregvanbrug) [inline lib/modules/@apostrophecms/ui/src/apos/components/AposAreaMenuItem.vue:51]: This should be able to be safely removed.

File: lib/modules/@apostrophecms/ui/src/apos/components/AposTag.vue [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+  methods: {
+    click() {
+      this.$emit('click', this.slug);
+    }
+  }
+};
+</script>
+
+<style lang="scss" scoped>
+// @import '../../scss/_mixins';
```
Comment (gregvanbrug) [inline lib/modules/@apostrophecms/ui/src/apos/components/AposTag.vue:35]: This should be able to be safely removed.

File: lib/modules/@apostrophecms/ui/src/apos/scss/_zindex.scss [hunk 4]
```diff
@@ -1,3 +1,4 @@
+$z-index-under: -1;
 $z-index-base: 0;
 $z-index-default: 1;
```
Comment (gregvanbrug) [inline lib/modules/@apostrophecms/ui/src/apos/scss/_zindex.scss:3]: Extremely minor but the default value for `z-index` corresponds to `0`. Maybe base should be called default and default should be something else?

File: lib/modules/@apostrophecms/ui/src/apos/components/AposAreaMenu.vue (hunk 1 above)
Comment (boutell) [reply lib/modules/@apostrophecms/ui/src/apos/components/AposAreaMenu.vue:184]: +1

File: lib/modules/@apostrophecms/ui/src/apos/components/AposAreaMenuItem.vue [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+    item: {
+      required: true,
+      type: Object
+    },
+    tabbable: {
+      type: Boolean,
+      default: true
+    }
+  },
+  emits: ['click', 'up', 'down'],
```
Comment (boutell) [inline lib/modules/@apostrophecms/ui/src/apos/components/AposAreaMenuItem.vue:32]: A Vue 3 ism, but doesn't seem to hurt in Vue 2, so let's start getting ready I guess!

Comment (abea) [reply lib/modules/@apostrophecms/ui/src/apos/components/AposAreaMenuItem.vue:32]: The linter yells about it. The alternative would be to change the rule to disable the requirement.

**Pull request merged**